internal_event_handlers.go

```diff
@@ -335,6 +335,40 @@ func (wc *workflowEnvironmentImpl) RequestCancelTimer(timerID string) {
 	wc.logger.Debug("RequestCancelTimer", zap.String(tagTimerID, timerID))
 }
 
+func validateVersion(changeID string, version, minSupported, maxSupported Version) {
```

Comment:
Can these methods return error message and panic at GetVersion() level? sometimes seeing too many panics in library code gives a feeling that panic is accepted pattern but we only do it in certain method calls, where error is not obvious choice.